### PR TITLE
remove hardcoded saferepr limit

### PR DESCRIPTION
**Pull request by ApaDoctor**
Py has hardcoded limit of saferepr. 
This PR fixes that.

### CODE REVIEW

File: py/_io/saferepr.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def _callhelper(self, call, x, *args):
                 s = s[:i] + '...' + s[len(s)-j:]
             return s
 
-def saferepr(obj, maxsize=240):
+def saferepr(obj, maxsize=240,maxother=160, maxarray=6, maxdict=4):
```
Comment (RonnyPfannschmidt) [inline py/_io/saferepr.py]: a space is missing here

File: py/_io/saferepr.py [hunk 2]
```diff
@@ -66,6 +66,9 @@ def saferepr(obj, maxsize=240):
     # review exception handling
     srepr = SafeRepr()
     srepr.maxstring = maxsize
-    srepr.maxsize = maxsize
-    srepr.maxother = 160
+    srepr.maxsize = srepr.maxlong = maxsize
+    srepr.maxother = maxother
+    srepr.maxdict = maxdict
+    srepr.maxlist = srepr.maxarray = srepr.maxset = srepr.maxmaxfrozenset = srepr.maxdeque = srepr.maxtuple = maxarray
```
Comment (RonnyPfannschmidt) [inline py/_io/saferepr.py]: i believe `maxmaxfrozenset` is a c&p mistake

**Comment by dimaspivak** [comment]
This would be very helpful for my team, as well. In our case, we're getting bitten by truncation in the asserts on lists. Anything I can do to help his one along as it looks to have stalled here?

**Comment by RonnyPfannschmidt** [comment]
@dimaspivak since @ApaDoctor doesnt seem to responds and its a simple one, its perhaps easiest to redo the pr

**Comment by RonnyPfannschmidt** [comment]
closing as no longer responsive,

**Pull request closed without merging**